graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/ProcessingLogType.java

```diff
@@ -20,5 +20,4 @@
 public enum ProcessingLogType implements ErrorClassification {
   Info,
   Warning,
-  ;
```

Comment:
There was a lot of these trailing semicolons. I wonder why?

Reply:
No idea..